[IMP] CRM: Add information about user portal to Reseller page (#18130)
odoo/documentation: content/applications/sales/crm/track_leads/resellers.rst

Reviewer: ```suggestion
the :guilabel:`Apps` application, and remove the :guilabel:`Apps` filter from the
```
Suggestion: Not sure if you need to use menuselect here since it's only one step. I replaced it with guilabel instead and removed 'application' from it so it wouldn't be bold.

Reviewer: ```suggestion
After the module is installed, navigate to the :menuselection:` CRM app --> Configuration`, there is a new section, titled :guilabel:`Resellers`, with three
```
Suggestion: Rewrote the sentence so the two menuselections are joined.

Reviewer: ```suggestion
With the Odoo **Website** and **Resellers** apps installed, a new webpage (`/partners`) is created to
display a list of all active partners from the **CRM** app.
```
Correction: Changed to bold

Reviewer: ```suggestion
the module and returns to the main Odoo dashboard.
```
Suggestion: removed comma

Reviewer: ```suggestion
New levels can be added, as needed, by clicking :guilabel:`New` and filling out the resulting level
```
Suggestion: removed comma

Reviewer: ```suggestion
from the list and make any desired changes on the level form page that appears.
```
Suggestion: Revised sentence.

Reviewer: ```suggestion
Level weight is used to decide the probability that a partner is assigned a lead or opportunity. On
```
Correction: Revised the sentence

Reviewer: ```suggestion
*Level Weight* can be assigned to an individual contact record. The weight assigned to the
individual record overrides the default weight set in the level configuration form.
```
Suggestion: revised sentences

Reviewer: ```suggestion
New partner activations can be added, as needed, by clicking :guilabel:`New` and entering a
```
Suggestion: removed comma

Reviewer: ```suggestion
the :guilabel:`Name` field of a desired level and enter a new name.
```
Suggestion: Removed comma

Reviewer: ```suggestion
<crm/partner-activations>` are configured.
```
Correction: added 'are'

Reviewer: ```suggestion
elements or information about the partner.
```
Suggestion: removed comma

Reviewer: ```suggestion
To use the reseller features, the **Resellers** module must be installed first. Navigate to
```
Suggestion: revised the sentence and bolded Resellers.

Reviewer: ```suggestion
Existing levels can also be edited and renamed if desired. To modify a level, select it
```
Suggestion: removed 'as well'

Reviewer: ```suggestion
Partner *activations* are used to determine a partner's status. Activations are assigned on an
```
Suggestion: revised sentence

Reviewer: ```suggestion
Three activation types are created by default in the **CRM** app:
```
Correction: changed to bold

Resellers
=========

Within Odoo's **CRM** app, leads can be forwarded to resellers (or partners). Leads can be manually
assigned or automatically assigned based on the resellers' designated *level* and location. Reseller
partners may also take assigned leads and convert them into opportunities in the user portal.

Configuration
=============

To use the reseller features, the **Resellers** module must be installed first. Navigate to the

> :guilabel:`Apps` application, and remove the :guilabel:`Apps` filter from the :guilabel:`Search...`

Reviewer 2: ```suggestion
:guilabel:`Apps` application, and remove the :guilabel:`Apps` filter from the search
```
updating from old standards (: 

might have to readjust the 100 char thing because of this change!

bar. Then, search for `Resellers`.

.. image:: resellers/resellers-module.png
   :alt: The resellers module in Odoo.

Click :guilabel:`Activate` on the :guilabel:`Resellers` module card that appears. Doing so installs
the module and returns to the main Odoo dashboard.

After the module is installed, navigate to the :menuselection:`CRM app`. Under the
After the module is installed, navigate to the :menuselection:` CRM app --> Configuration`, there is
a new section, titled :guilabel:`Resellers`, with three options beneath it: :guilabel:`Partner

> Levels`, :guilabel:`Partner Activations`, and :guilabel:`Commission Plans`.

Reviewer 2: Optional: this feels a little bit like a run-on to me. How about:

```suggestion
After the module is installed, navigate to :menuselection:` CRM app --> Configuration`. There is
a new section, titled :guilabel:`Resellers`, with three options beneath it: :guilabel:`Partner
Levels`, :guilabel:`Partner Activations`, and :guilabel:`Commission Plans`.
```

> Within Odoo's *CRM* app, leads can be forwarded to resellers (or partners). Leads can be manually

Reviewer: ```suggestion
Within Odoo's **CRM** app, leads can be forwarded to resellers (or partners). Leads can be manually
```

> assigned, or automatically assigned, based on the resellers' designated *level* and location.

Reviewer: ```suggestion
assigned or automatically assigned based on the resellers' designated *level* and location.
```
Suggestion: removed commas

- :guilabel:`Silver`

> - :guilabel:`Bronze`

Reviewer 2: when describing default behavior, ensure this is default behavior of newly-created DBs, not in runbot. Runbots contain demo data that customers wouldn't see, so describing demo data default behavior would be confusing to them.

if you decide to keep this section in the doc, can you please confirm in a newly created DB that these partner levels are avialable by default? :D

Author: Confirmed default behavior in a base runbot with just CRM and Resellers module.

===================

After :ref:`partner levels <crm/partner-levels>` and :ref:`partner activations

> <crm/partner-activations>` are configured.

Reviewer 2: it seems the original sentence was an incomplete thought: after doing x, ... ? perhaps we can add:

```suggestion
<crm/partner-activations>` are configured, assign partners to [perhaps elaborate why we are doing this?]
```

Failing to fill out the form **prevents** the lead from converting to an opportunity.

.. image:: resellers/lead-feedback-popup.png
   :alt: The Lead Feedback form brought up by clicking the "I'm interested" button.

If the partner is not interested in the opportunity, the :guilabel:`Lead Feedback` form prompts the
partner to enter why they're not interested in the lead, whether they've contacted the customer, and
whether they think the lead is spam. This version of the :guilabel:`Lead Feedback` form does not
need to be completed to be confirmed.

.. image:: resellers/lead-feedback-popup-not-interested.png
   :alt: The Lead Feedback form brought up by clicking the "I'm not interested" button.

Partner opportunities
---------------------

Leads accepted by the partner are converted into *opportunities*. Clicking an opportunity reveals
the same information displayed in the **CRM** app, but in a view designed for the user portal. From
an opportunity's page, the reseller may:

* Change the opportunity's stage.
* Edit the opportunity's information, including priority, planned revenue, and pertinent dates.
* Update the contact information for the opportunity.

> * Leave notes, messages, and more in the opportunity's chatter.

Reviewer 2: while the `*` works for bullet points, let's stick to the `-` since that's how we format the other bullet points in other articles!

```suggestion
- Change the opportunity's stage.
- Edit the opportunity's information, including priority, planned revenue, and pertinent dates.
- Update the contact information for the opportunity.
- Leave notes, messages, and more in the opportunity's chatter.
```

Repeat these steps for all partners.

.. image:: resellers/partners-webpage.png
   :alt: An example of the partners webpage, displaying available partners by level and location.

Partners and the user portal
============================

Reseller partners who have been assigned leads and opportunities can view them in their user
portals.

.. image:: resellers/lead-opportunity-buttons.png

>    :alt: An example of the partners' user portal and the lead and opportunity buttons.

Reviewer: Suggestion: Move to the end of the section. That way, the text isn't broken up by the image.

Author: Good note!

Clicking either the :guilabel:`Leads` or :guilabel:`Opportunities` button brings up a list of all
leads or opportunities assigned to that partner. Individual leads and opportunities can be clicked
for more details.

Partner leads
-------------

Leads viewed through a partner's user portal contain the same information as the lead viewed through
the **CRM** app. Reseller partners may click either the :guilabel:`I'm interested` or :guilabel:`I'm
not interested` buttons. Clicking either button brings up the :guilabel:`Lead Feedback` form.

.. image:: resellers/reseller-lead-page.png
   :alt: The lead page as seen in a reseller's user portal.

If the partner has expressed interest, the form prompts them to enter their next planned action and
expected revenue for the lead and to confirm that they have contacted the customer. Entering this
information and clicking the :guilabel:`Confirm` button converts the lead into an opportunity.

> Failing to fill out the form prevents the lead from being able to convert to an opportunity.

Reviewer: ```suggestion
Failing to fill out the form **prevents** the lead from converting to an opportunity.
```
Suggestion: This information seems important enough to highlight more.

.. image:: resellers/lead-feedback-popup.png
   :alt: The Lead Feedback form brought up by clicking the "I'm interested" button.

If the partner is not interested in the opportunity, the :guilabel:`Lead Feedback` form prompts the
partner to enter why they're not interested in the lead, if they've contacted the customer, and if

> they think the lead is spam. This version of the :guilabel:`Lead Feedback` form does not need to be

Reviewer: ```suggestion
partner to enter why they're not interested in the lead, whether they've contacted the customer, and whether
they think the lead is spam. This version of the :guilabel:`Lead Feedback` form does not need to be
```
Suggestion

completed to be confirmed.

.. image:: resellers/lead-feedback-popup-not-interested.png
   :alt: The Lead Feedback form brought up by clicking the "I'm not interested" button.

Partner opportunities
---------------------

Leads that have been accepted by the partner are converted into *opportunities*. Clicking into an
opportunity reveals the same information as is displayed in the **CRM** app but in a view that's

> designed for the user portal. From an opportunity's page, the reseller may:

Reviewer: ```suggestion
Leads accepted by the partner are converted into *opportunities*. Clicking an
opportunity reveals the same information displayed in the **CRM** app, but in a view
designed for the user portal. From an opportunity's page, the reseller may:
```
Suggestion

* Change the opportunity's stage
* Edit the opportunity's information, including priority, planned revenue, and pertinent dates
* Update the contact information for the opportunity

> * Leave notes, messages, and more in the opportunity's chatter

Reviewer: ```suggestion
* Change the opportunity's stage.
* Edit the opportunity's information, including priority, planned revenue, and pertinent dates.
* Update the contact information for the opportunity.
* Leave notes, messages, and more in the opportunity's chatter.
```
Suggestion: added end punctuation

.. note::
   The reseller's ability to actually change these settings depends on the access that has been
   granted to them by a database administrator. See the ::doc:`access rights

>    <../../../general/users/access_rights>` documentation for more details.

Reviewer: ```suggestion
   The reseller's ability to change these settings depends on the access
   granted to them by a database administrator. See the ::doc:`access rights
   <../../../general/users/access_rights>` documentation for more details.
```
Suggestion
